<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wienwahl</class>
 <widget class="QMainWindow" name="Wienwahl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wienwahl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>831</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAddNewRow"/>
    <addaction name="actionDuplicateRow"/>
    <addaction name="actionDeleteRow"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionSaveToDB"/>
    <addaction name="actionLoadFromDB"/>
    <addaction name="separator"/>
    <addaction name="actionGenerateProjection"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDatabase"/>
  </widget>
  <action name="actionNeu_STRG_N">
   <property name="text">
    <string>Neu... [STRG+N]</string>
   </property>
  </action>
  <action name="action_ffnen_STRG_O">
   <property name="text">
    <string>Ã–ffnen... [STRG+O]</string>
   </property>
  </action>
  <action name="actionSave_STRG_S">
   <property name="text">
    <string>Save... [STRG+S]</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAddNewRow">
   <property name="text">
    <string>Add new row</string>
   </property>
  </action>
  <action name="actionDuplicateRow">
   <property name="text">
    <string>Duplicate row</string>
   </property>
  </action>
  <action name="actionDeleteRow">
   <property name="text">
    <string>Delete row</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionSaveToDB">
   <property name="text">
    <string>Save to DB</string>
   </property>
  </action>
  <action name="actionLoadFromDB">
   <property name="text">
    <string>Load from DB</string>
   </property>
  </action>
  <action name="actionGenerateProjection">
   <property name="text">
    <string>Generate projection</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
